### DISCUSSION TITLE
TurboWarp file/url upload without sandbox

### DISCUSSION THREAD

**Discussion by Eaielectronic** [TurboWarp/extensions#1706] [Ideas]
It would be nice to be able to load extensions from GitHub without the sandbox to be able to update them (a box like with files) and also add the fact that if we reimport an extension in custom extension having the same id as one already registered, it overwrites it to be able to update it like TurboWarp extensions (when it overwrites, it should not delete the blocks in the extension project and display an alert message if the new extension does not include a block that is in the old version). Many people ask for this option.

**Comment by LilyMakesThings** [comment]
you can already update extensions; if you load the project after loading the new version of the custom extension, it'll replace it automatically

**Comment by Eaielectronic** [comment]
It would really be necessary to add a box without sandbox for the urls and to update via the text or the file it does not work or very rarely. It would be necessary to be able to update them without problem.

**Reply by LilyMakesThings** [reply]
I explained the intended way to do it and I can't replicate any "problems" on my end. As for URLs, forcing URLs to be run in a sandbox is a security measure.

**Comment by Eaielectronic** [comment]
you should be able to add your own domain name to add extensions without sandbox (for example I put my domain name which is https://Eaielectronic.github.io and all the extensions having domain name are loaded without sandbox.)

**Comment by Eaielectronic** [comment]
So if you put your domain name with github in it, it should be more secure...
